src/app/lists/ListDataview.tsx

```diff
@@ -183,17 +183,24 @@ export const SnackBarDiv = styled.div`
 `;
 
 const TableDiv = styled.div`
-    margin: 20px;
+    margin: 40px;
     padding: 20px;
-    background-color: ${(props) => props.theme.surfaceBackgroundColor};
-    border: solid 1px ${(props) => props.theme.surfaceForegroundColor};
+    background-color: ${(props) => props.theme.main.background[0]};
+    box-shadow: 0 0 15px ${(props) => props.theme.shadow};
     border-radius: 1rem;
 `;
 
 const StyledAddButton = styled.button`
```

Comment:
Is it worth changing this to an `MuiButton` for a more consistent look?

Reply:
Sure
For consistency I changed the clear button too, and restyled input and accordian to have the same height and font size